server/routes/url.js

```diff
@@ -1,11 +1,12 @@
 const express = require("express");
 const { getSpecificUrl, createUrl } = require("../controller/urls");
+const { rateLimit } = require("../controller/rateLimit");
 
 const urlRouter = new express.Router();
 
 //This route is used to redirect to original url using short url as id
-urlRouter.get("/:shortId", getSpecificUrl);
+urlRouter.get("/:shortId", rateLimit, getSpecificUrl);
```

Comment:
remove rate limit from here also users can see there shortened URL else it will not open after certain limit.

Reply:
I think we can change the no of count and time 

I understand that it will not open after a certain limit. 

because all the routes need to be secure by a certain limit right?

Reply:
Then it's ok

I remove the middleware that I made 

we can add it once we get a lot of requests

Reply:
Just remove from routes as we still need for creating this